src/pygeoprocessing/routing/routing.pyx

```diff
@@ -77,12 +77,12 @@ cdef double SQRT2_INV = 1.0 / 1.4142135623730951
 #  321
 #  4x0
 #  567
-cdef int *D8_XOFFSET = [1, 1, 0, -1, -1, -1, 0, 1]
-cdef int *D8_YOFFSET = [0, -1, -1, -1, 0, +1, +1, +1]
+cdef int * D8_XOFFSET = [1, 1, 0, -1, -1, -1, 0, 1]
+cdef int * D8_YOFFSET = [0, -1, -1, -1, 0, +1, +1, +1]
 
 # this is used to calculate the opposite D8 direction interpreting the index
 # as a D8 direction
-cdef int* D8_REVERSE_DIRECTION = [4, 5, 6, 7, 0, 1, 2, 3]
+cdef int * D8_REVERSE_DIRECTION = [4, 5, 6, 7, 0, 1, 2, 3]

```

Comment:
Ahh I'm guessing it was inconsistent spacing. I'd prefer the * on the variable name rather than the type.